build_tools/github_actions/configure_ci.py

```diff
@@ -10,20 +10,27 @@
 The following environment variables are required:
 - GITHUB_EVENT_NAME: GitHub event name, e.g. pull_request.
 - GITHUB_OUTPUT: path to write workflow output variables.
+- GITHUB_STEP_SUMMARY: path to write workflow summary output.
 
 When GITHUB_EVENT_NAME is "pull_request", there are additional environment
 variables to be set:
 - PR_TITLE (required): PR title.
 - PR_BODY (optional): PR description.
 - BASE_REF (required): base commit SHA of the PR.
+- ORIGIN_PR_TITLE (optional): PR title from the original PR event, showing a
+    notice if PR_TITLE is different.
+- ORIGIN_PR_BODY (optional): PR description from the original PR event, showing
+    a notice if PR_BODY is different. ORIGIN_PR_TITLE must also be set.
```

Comment:
Spell out ORIGINAL in these vars? Two extra characters seems reasonable for clarity